upgrade/Upgrade-6.4.3.php

```diff
@@ -74,18 +75,10 @@ function upgrade_module_6_4_3($module)
     ];
 
     foreach ($dirs as $dir) {
-        if (file_exists($baseDir . $dir)) {
-            rmdir($baseDir . $dir);
-        }
+        $kit->rrmdir($baseDir . $dir);
```

Comment:
Rrmdir ? Sure with double r to start ?

Reply:
Yes, it is not a typo. In fact, that code is going to be extended in PR: https://github.com/202ecommerce/paypal/pull/359